Expired/Views/ProductListView.swift

```diff
@@ -18,8 +18,14 @@ struct ProductListView: View {
                 if filteredProducts.count > 0 {
                     List {
                         ForEach(filteredProducts) { product in
-                            ProductCell(product: product)
-                        }
+                            NavigationLink {
+                                ProductEditView(product: product)
+                                    .environmentObject(productStore)
+                                }label:{
+                                    ProductCell(product: product)
+                                    .environmentObject(productStore)
+                                }
+                            }
```

Comment:
```suggestion
                            NavigationLink {
                                ProductEditView(product: product)
                            } label: {
                                ProductCell(product: product)
                            }
```

- Pay attention to indentation and spacing.
- `environmentObject` is not required, as it's already attached to the root view and available in any descendent views.